tests/pillar/test-linux-main/main.sls

```diff
@@ -1,3 +1,3 @@
 nessus-agent:
   lookup:
-    package_url_es6: https://S3BUCKET.F.Q.D.N/mcafee/linux/nessus-agent/NessusAgent.rpm
+    package_url: https://S3BUCKET.F.Q.D.N/mcafee/linux/nessus-agent/NessusAgent.rpm
```

Comment:
Maybe while we're in here also change `mcafee` => `nessus`?

Reply:
Done!